### DISCUSSION TITLE
Failed to find a valid digest in the 'integrity' attribute for resource

### DISCUSSION THREAD

**Discussion by Smiog** [razonyang/hugo-theme-bootstrap#355] [Q&A] [answered]
#### Anyone able to solve?
-------------
#### 有人会解决吗？

**Comment by razonyang** [comment]
Could you please provide the information of your browser console? For example, press `F12` in Chrome.

**Reply by Smiog** [reply]
Of course.
![image](https://user-images.githubusercontent.com/90970249/150634219-c828064c-e81b-4627-8206-d596628151b3.png)
Is this what you wanted?

**Reply by razonyang** [reply]
See also https://github.com/razonyang/hugo-theme-bootstrap/issues/209#issuecomment-925029489.

**Reply by Smiog** [reply]
@razonyang Sorry but it didn't work well neither after I executed the commands. Maybe I made a mistake?
![image](https://user-images.githubusercontent.com/90970249/150635298-7bd8cfd3-bd34-4a77-b3d3-3c6008a5bf14.png)
It seems that it has changed nothing  :(

**Reply by razonyang** [reply]
Could you please provide your build environment? Such as OS, hugo version... So that I can reproduce this issue.

**Reply by Smiog** [reply]
__OS__ ：`Win11`

__hugo__ ：`v0.91.2`

### Ty very much!

**Comment by Smiog** [comment, marked as answer]
@razonyang I think this problem is because of GitHub. Do you think it would be better off deploying to another code-hosting platform? Such as GitLab, Gitee, Coding, etc.

**Reply by razonyang** [reply]
@SmileZxk I can't reproduce the issue. I didn't use other git platforms for hosting a static site, so I can't provide useful advice.

**Reply by Smiog** [reply]
> @SmileZxk I can't reproduce the issue. I didn't use other git platforms for hosting a static site, so I can't provide useful advice.

You made your own server to host your site, right? May you teach me how to do it? Ty.

**Reply by razonyang** [reply]
It's very hard to make it clear in few words. You need additional expenses to buy servers and domain names, and then set up a web server and deploy your site on it, there are also a lot of stuff need to do, such as HTTPS certifications, CDN. So I recommend using others git platforms for hosting if you're not similar to set up a self-hosted server.

[Netlify](https://www.netlify.com/) may be worth a try, although it's slow in China.

**Comment by Smiog** [comment]
# Thx!!!
## I use netlify and my site has been running perfectly!!
